com.sublimemerge.App.yaml

```diff
@@ -16,6 +16,7 @@ finish-args:
   - --share=network
   - --filesystem=host  # uses the SDK's git binary directly, no portals
   - --own-name=com.sublimemerge  # app doesn't follow the app ID spec
+  - --env=PATH=/app/bin:/usr/bin:/app/libexec/git-lfs
```

Comment:
I would suggest putting this in the `sublime_merge.sh` script below as well, so that you can append to the path, e.g. `export PATH="$PATH:/app/libexec/git-lfs"`. I think that way it will still work if someone tries to override `--env=PATH` on the command line.